src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -13,6 +15,8 @@
   left: 0;
   z-index: 1;
   pointer-events: auto;
+  position: relative;
+  padding: 0 $gui-gutter 4px;
```

Comment:
position and padding should be added to the top bar as well?

Reply:
done